userguide/services.rst

```diff
@@ -1456,6 +1456,11 @@ screen is really a front-end to
    | Description                 | string            | Enter an optional server description.                                                                |
    |                             |                   |                                                                                                      |
    +-----------------------------+-------------------+------------------------------------------------------------------------------------------------------+
+   | Enable SMB1 support         | checkbox          | Allow legacy SMB clients to connect to the server. **Warning:** SMB1 has been deprecated as it is    |
```

Comment:
This should be more specific: "SMB1 is not secure and has been deprecated by Microsoft."